modules/custom/az_cas/config/install/cas.settings.yml

```diff
@@ -0,0 +1,52 @@
+server:
+  version: '2.0'
+  protocol: https
+  hostname: webauth.arizona.edu
+  port: 443
+  path: /webauth
+  verify: 0
+  cert: ''
+gateway:
+  check_frequency: -2
+  paths:
+    id: request_path
+    pages: ''
+    negate: false
+forced_login:
+  enabled: false
+  paths:
+    id: request_path
+    pages: ''
+    negate: false
+logout:
+  logout_destination: '<front>'
+  enable_single_logout: false
+  cas_logout: true
+  single_logout_session_lifetime: 25
+proxy:
+  initialize: false
+  can_be_proxied: false
+  proxy_chains: ''
+user_accounts:
+  prevent_normal_login: true
+  auto_register: false
+  email_assignment_strategy: 0
+  email_hostname: ''
```

Comment:
In D7 we used `email.arizona.edu` for this.

Reply:
D8 should be arizona.edu

Reply:
Is it just arizona.edu now or does it still need the email. part?

Reply:
I think we can just use `arizona.edu` now.

Reply:
You can now use just @arizona.edu as far as I know.